iOSClient/AppDelegate.swift

```diff
@@ -168,7 +169,9 @@ class AppDelegate: UIResponder, UIApplicationDelegate, UNUserNotificationCenterD
         self.presentPasscode {
             self.enableTouchFaceID()
         }
-
+        adjust.configAdjust()
```

Comment:
check if this can be moved to single function to init adjust. May be like inside AdjustHelper.
By doing this only 1 function call will be made instead of two.